compliance_checker/tests/test_suite.py

```diff
@@ -184,3 +197,12 @@ def test_standard_output_score_header(self):
                                                         groups)
         assert all_passed < out_of
 
+    def test_netCDF4_features(self):
+        """
+        Check if a proper netCDF4 file with netCDF4-datatypes is created.
+        """
+        # create and open dataset
+        ds = self.cs.load_dataset(static_files['netCDF4'])
+        # check if types are correct
+        assert isinstance(ds.global_att_of_type_int, np.int32)
+        assert isinstance(ds.global_att_of_type_int64, np.int64)
```

Comment:
While you're already doing type-checking for global attributes, it probably wouldn't hurt to type-check the actual variables as well.

Reply:
This was mainly to check whether the netCDF4 type `int64` is properly used. Should I do the same for a variable?